src/app/tournois/page.tsx

```diff
@@ -0,0 +1,36 @@
+'use client'
+
+import Link from 'next/link'
+import { useEffect, useState } from 'react'
+
+import { TournamentTile } from '../../components/TournamentTile'
+import { usePocketBase } from '../../contexts/PocketBaseContext'
+import type { Tournament } from '../../models/Tournament'
+
+const TournamentSuscription: React.FC = () => {
+  const { pb } = usePocketBase()
+  const [tournaments, setTournaments] = useState<Tournament[]>([])
+
+  useEffect(() => {
+    pb.collection('tournaments')
+      .getFullList<Tournament>({
+        sort: '-created',
+      })
+      .then(res => {
+        setTournaments(res)
+      })
```

Comment:
C'est un peu pour pinailler, le change pas si tu veux pas, mais tu peux faire :
```suggestion
      .then(setTournaments)
```